db/migrate/20220310140825_create_registration_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRegistrationTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :registrations do |t|
+      t.uuid :reference
+      t.boolean :is_transferred
```

Comment:
Maybe set this to a datetime instead of a boolean so we can see when it was transferred

Reply:
Yeah will do